packages/itwin/tree-widget/src/components/trees/external-sources-tree/ExternalSources.json

```diff
@@ -0,0 +1,230 @@
+{
+  "$schema": "../../../../node_modules/@itwin/presentation-common/Ruleset.schema.json",
+  "id": "ExternalSources",
```

Comment:
should probably use the same prefix as the other tree rulesets?

tree-widget-react:ExternalSources

Reply:
Updated ruleset id to match others